Classes/Backend/Base/Model/KalturaAccessControlMessage.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  KalturaAccessControlMessage.swift
+//  Pods
+//
+//  Created by Rivka Peleg on 05/07/2017.
+//
+//
+
```

Comment:
@noamtamim  let's make sure this PR is not merged without your script for headers.

Thanks :)